app/src/main/res/layout-sw600dp-land/question_player_fragment.xml

```diff
@@ -178,9 +174,9 @@
         android:visibility="invisible" />
 
       <nl.dionsegijn.konfetti.KonfettiView
-        android:id="@+id/banner_confetti_view"
+        android:id="@+id/congratulations_text_confetti_view"
         android:layout_width="match_parent"
-        android:layout_height="match_parent"/>
+        android:layout_height="132dp"/>
```

Comment:
Here & elsewhere: thanks for making this change--I find this much clearer than the fixed-size FrameLayout from before.